.gitignore

```diff
@@ -13,3 +13,6 @@ node_modules
 # TS Build output logs
 *.tsbuildinfo
 
+# Downloads
+/downloads
+
```

Comment:
In the more-specific `e2e/.gitignore`-file this already seems to be ignored...

Is it necessary to ignore files in: `121-platform/downloads/*` or in `121-platform/e2e/downloads/*`?

The overall approach of the whole repo is to put this type of configuration "as close as possible" to the actual use-cases. (So in this case; inside the `e2e`-folder.)

Reply:
I'll remove it seems like I used wrong download path